_posts/en/newsletters/2019-03-26-newsletter.md

```diff
@@ -0,0 +1,256 @@
+  the onchain payment contains a refund condition that allows Bob to
+  spend it back to himself after a timelock expires.
+
+    Most of the process is trustless, so neither party has an
+    opportunity to steal from the other (provided the software operates
+    (and is operated) correctly).  The exception is the creation of the
+    initial onchain transaction and the possible need for Bob to create
+    a refund transaction: if the trustless exchange doesn't happen, Bob
+    will receive no compensation for the onchain transaction fees
+    required for both of those transactions.  According to the [Loop
+    documentation][], their implementation has Alice send Bob a small
+    trusted payment in advance of the trustless exchange as an act of
+    good faith and an assurance that the operation won't end up costing
+    Bob money.
+
+    By allowing Alice and Bob to swap onchain and offchain funds, all
+    while continuing to use their existing channels, Loop helps users
+    keep their channels open longer and makes it conceivable that they
+    could stay open indefinitely.
+
+- **Square Crypto developer group announced:** the CEO of Square
+  [announced on Twitter][sqcrypto announced] that they are forming a
+  group to employ several contributors to open source Bitcoin projects,
+  including both developers and a designer.  See their announcement for
+  application instructions.  (Note: Square is also a sponsoring member
+  of Optech.)
+
+## Bech32 sending support
+
+*Week 2 of 24.  From now until the second anniversary of the segwit soft
+fork lock-in on 24 August 2019, the Optech Newsletter will contain this
+weekly section that provides information to help developers and
+organizations implement bech32 sending support---the ability to pay
+native segwit addresses.  This [doesn't require implementing
+segwit][bech32 easy] yourself, but it does allow the people you pay to
+access all of segwit's multiple benefits.*
+
+As described [last week][bech32 easy], implementing just segwit
+spending should be easy.  Yet we suspect some managers might wonder
+whether there are enough people using segwit to justify their team
+spending development effort on it.  This week, we look at sites that
+track various segwit adoption statistics so that you can decide whether
+it's popular enough that your wallet or service might become an outlier
+by failing to support it soon.
+
+Optech tracks statistics about segwit use on our [dashboard][optech
+dashboard]; another site tracking related statistics is [P2SH.info][].
+We see an average of about 200 outputs per block are sent to native
+segwit addresses.  Those outputs are then spent in about 10% of all
+Bitcoin transactions.  That makes payments to native segwit addresses
+more popular than almost all altcoins.
+
+![Screenshot of Optech Dashboard segwit usage stats](/img/posts/2019-03-segwit-usage.png)
+
+However, wallets supporting segwit can also receive payment to
```

Comment:
I think this paragraph would be a little confusing for people not familiar with P2SH-wrapped segwit. Do you think the following would be clearer:

```
However, wallets supporting segwit can also receive payment to
backwards-compatible _P2SH-wrapped segwit_ addresses.  We can't directly track how many P2SH address are P2SH-wrapped segwit, but
the statistics sites linked earlier do tell us how often users spend
bitcoins they received to a P2SH-wrapped segwit address.  Currently, that
averages at about 37% of transactions.  If adoption remained steady at
that level, it would represent a minimum average of 1,400 outputs per
block.  Likely, it's even higher now.
```

I don't understand what you mean by '37% of transactions. Are you saying that 37% of spends are from P2SH-wrapped segwit UTXOs? Or that 37% of P2SH outputs that are spent are P2SH-wrapped segwit?

Reply:
I told @jnewbery wrong.  It's that 37% of transactions contain at least one input that spends from a nested-segwit output.

I'll be editing the paragraph to try to improve clarity.